test/test_contact_importer.rb

```diff
@@ -20,10 +15,10 @@ def test_u_p_import
   end
 
   def test_aol_import
-    VCR.use_cassette("aol_import", :match_requests_on => [:uri, :host, :path]) do
+    VCR.use_cassette("aol_import", :match_requests_on => [:method, :host, :path]) do
       importer = Cloudsponge::ContactImporter.new(DOMAIN_KEY, DOMAIN_PASSWORD)
       resp = importer.begin_import('AOL')
-      if resp[:consent_url]
+      if ENV['request_consent'] && resp[:consent_url]
```

Comment:
Nervermind, I get it, you can set in the tests to request consent or not, thanks!